src/main/java/data/domain/Name.java

```diff
@@ -0,0 +1,23 @@
+package data.domain;
+
+import static data.domain.RacingGameConstants.NAME_MAX_LENGTH;
+
+public class Name {
+
+    private final String name;
+
+    private Name(String name) {
+        if (name.length() > NAME_MAX_LENGTH) {
```

Comment:
```suggestion
        if (name == null || name.length() > NAME_MAX_LENGTH) {
```

null 체크를 추가하거나, apache commons 같은 String 관련 util 을 사용하는 것도 좋을 것 같습니다. 🙇‍♂️